.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- choose all that apply -->
+**Type of issue**: bug report | feature request | question | other enhancement
+
+<!-- if the issue is NOT a question, fill out the following sections -->
+
+<!-- choose one -->
+**Impact**: no rtl change | rtl refactoring | new rtl | unknown
```

Comment:
Chipyard is not BOOM. Not everything is RTL.
I think there is a low likelihood of someone filing an issue and knowing the impact ahead of time

Reply:
I think RTL changes should be called out somehow though.